src/main/java/dao/StorageDao.java

```diff
@@ -0,0 +1,13 @@
+package dao;
+
+import model.FruitModel;
+
+public interface StorageDao {
+    int get(String key);
+
+    boolean put(FruitModel fruitModel);
+
+    boolean replace(String name, int amount);
```

Comment:
It is better to rename this method to update to do it according to the CRUD method naming principle